internal/cfgplugins/networkinstance.go

```diff
@@ -68,10 +69,30 @@ func ConfigureNetworkInstance(t *testing.T, dut *ondatra.DUTDevice, netInstName
 	return ni
 }
 
+// UpdateNetworkInstanceOnDut updates the network instance on the DUT. Not using batch update
 func UpdateNetworkInstanceOnDut(t *testing.T, dut *ondatra.DUTDevice, netInstName string, netInst *oc.NetworkInstance) {
 	gnmi.Update(t, dut, gnmi.OC().NetworkInstance(netInstName).Config(), netInst)
 }
 
+// ConfigureDefaultNetworkInstance configures the default network instance name and type.
+func ConfigureDefaultNetworkInstance(batch *gnmi.SetBatch, t testing.TB, d *ondatra.DUTDevice) {
```

Comment:
Here and throughout, follow style guide, use "New" as the prefix, instead of "Configure"

```suggestion
func NewDefaultNetworkInstance(batch *gnmi.SetBatch, t testing.TB, d *ondatra.DUTDevice) {
```